aiidalab/appstore.py

```diff
@@ -115,36 +115,17 @@ def change_vis_list(self, _=None):
 
     def render(self, _=None):
         """Show all the available apps."""
-        max_len = 180
         self.output.clear_output()
         page = self.page_selector.value - 1
         start = page * self.items_per_page.value
         end = (page + 1) * self.items_per_page.value
         with self.output:
-            horisontal_line = ipw.HTML('<hr>')
             for number, app_base in enumerate(self.apps_to_display[start:end]):
                 if self.category_filter.value and self.app_corresponding_categories[start:end][number]:
                     display(ipw.HTML("<h1>{}</h1>".format(
                         self.app_corresponding_categories[start:end][number].title())))
 
-                description = ipw.HTML("""<h2 style="text-align:center;">{}</h2>
-                <div style="font-size: 15px; font-style: italic">Description: {}</div>
-                <br>
-                <div style="text-align:right;">{}</div>
-                """.format(
-                    app_base.title,
-                    app_base.description if len(app_base.description) < max_len else app_base.description[:max_len] +
-                    '...', app_base.more))
-                description.layout = {'width': '600px'}
-                description.layout.margin = "0px 0px 0px 40px"
-
-                # Warning! put here as less widgets as possible. They take a lot of time to load.
-                result = ipw.VBox([
-                    horisontal_line,
-                    ipw.HBox([app_base.logo, description]),
-                    ipw.HBox([app_base.uninstall_button, app_base.update_button, app_base.install_button]),
-                    ipw.HBox([app_base.install_info]),
-                ])
-                display(result)
+                widget = AppManagerWidget(app_base)
```

Comment:
I think it is a good idea to have a separate widget that defines app appearance, but would it be possible to make it more customizable. I don't thing the app should look exactly the same in the App store and in 'Manage App' page

Reply:
I guess I kind of ran out of steam there. I can reintroduce the different representation within the different contexts.

Reply:
great.